nym-wallet/src/pages/bonding/node-settings/settings-pages/general-settings/ParametersSettings.tsx

```diff
@@ -0,0 +1,214 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+  Button,
+  Divider,
+  Typography,
+  TextField,
+  InputAdornment,
+  Grid,
+  Alert,
+  IconButton,
+  CircularProgress,
+} from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CloseIcon from '@mui/icons-material/Close';
+import { isMixnode } from 'src/types';
+import { updateMixnodeCostParams } from 'src/requests';
+import { TBondedMixnode, TBondedGateway } from 'src/context/bonding';
+import { SimpleModal } from 'src/components/Modals/SimpleModal';
+import { bondedNodeParametersValidationSchema } from 'src/components/Bonding/forms/mixnodeValidationSchema';
+
+export const ParametersSettings = ({ bondedNode }: { bondedNode: TBondedMixnode | TBondedGateway }): JSX.Element => {
+  const [open, setOpen] = useState(true);
+  const [openConfirmationModal, setOpenConfirmationModal] = useState<boolean>(false);
+
+  const theme = useTheme();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting, isDirty, isValid },
+  } = useForm({
+    resolver: yupResolver(bondedNodeParametersValidationSchema),
+    mode: 'onChange',
+    defaultValues: isMixnode(bondedNode)
+      ? {
+          operatorCost: bondedNode.bond.amount,
+          profitMargin: bondedNode.profitMargin,
+        }
+      : {},
+  });
+
+  const onSubmit = async (data: { operatorCost?: string; profitMargin?: string }) => {
+    if (data.operatorCost && data.profitMargin) {
+      const MixNodeCostParams = {
+        profit_margin_percent: data.profitMargin.toString(),
+        interval_operating_cost: {
+          denom: bondedNode.bond.denom,
+          amount: data.operatorCost.toString(),
+        },
+      };
+      try {
+        await updateMixnodeCostParams(MixNodeCostParams);
+        setOpenConfirmationModal(true);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+  };
+
+  return (
+    <Grid container xs item>
+      {open && (
+        <Alert
+          severity="info"
+          action={
+            <IconButton
+              aria-label="close"
+              color="inherit"
+              size="small"
+              onClick={() => {
+                setOpen(false);
+              }}
+            >
+              <CloseIcon fontSize="inherit" />
+            </IconButton>
+          }
+          sx={{
+            width: 1,
+            px: 2,
+            borderRadius: 0,
+            bgcolor: 'background.default',
+            color: (theme) => theme.palette.nym.nymWallet.text.blue,
+            '& .MuiAlert-icon': { color: (theme) => theme.palette.nym.nymWallet.text.blue, mr: 1 },
+          }}
+        >
+          <strong>Profit margin can be changed once a month, your changes will be applied in the next interval</strong>
```

Comment:
We should avoid to use "raw" HTML tags like `strong` `span` `div` etc. You'd might use the MUI equivalent ones to keep consistency.